tests/Polkadot/AddressTests.cpp

```diff
@@ -46,6 +46,6 @@ TEST(PolkadotAddress, FromPublicKey) {
 }
 
 TEST(PolkadotAddress, FromString) {
-    auto address = Address("15KRsCq9LLNmCxNFhGk55s5bEyazKefunDxUH24GFZwsTxyu");
-    ASSERT_EQ(address.string(), "15KRsCq9LLNmCxNFhGk55s5bEyazKefunDxUH24GFZwsTxyu");
+        auto address = Address("15KRsCq9LLNmCxNFhGk55s5bEyazKefunDxUH24GFZwsTxyu");
+        ASSERT_EQ(address.string(), "15KRsCq9LLNmCxNFhGk55s5bEyazKefunDxUH24GFZwsTxyu");
```

Comment:
```suggestion
    auto address = Address("15KRsCq9LLNmCxNFhGk55s5bEyazKefunDxUH24GFZwsTxyu");
    ASSERT_EQ(address.string(), "15KRsCq9LLNmCxNFhGk55s5bEyazKefunDxUH24GFZwsTxyu");
```

Reply:
Only one half of the 'git mv' got checked in, fixed.
I moved SS58Address from generic `src` to Polkadot, as it is used only by Polkadot and Kusama.

Reply:
These indentions still looks weird